apps/wakunode2/wakunode2.nim

```diff
@@ -286,45 +286,19 @@ proc initNode(conf: WakuNodeConf,
                         dnsResolver,
                         conf.relayPeerExchange, # We send our own signed peer record when peer exchange enabled
                         dns4DomainName,
+                        conf.discv5Discovery,
+                        DiscoveryConfig.init(conf.discv5TableIpLimit,
+                                             conf.discv5BucketIpLimit,
+                                             conf.discv5BitsPerHop),
```

Comment:
May be more readable to assign this to a variable before calling `.new()`

Reply:
Addressed in [63878f0](https://github.com/waku-org/nwaku/pull/1512/commits/63878f0b5d3d554a36e64368ba8b9c67d495ce8d)